packages/vm/src/runTx.ts

```diff
@@ -207,6 +210,20 @@ export async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
 async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
   const state = this.stateManager
 
+  let stateAccesses
+  if (this.common.isActivatedEIP(6800)) {
+    if (!(this.stateManager instanceof StatelessVerkleStateManager)) {
+      throw Error(`StatelessVerkleStateManager needed for execution of verkle blocks`)
+    }
+    stateAccesses = (this.stateManager as StatelessVerkleStateManager).accessWitness
+  } else {
+    // assign an empty witness to prevent unnecessary code spagetti of null/undefined checks while usage
```

Comment:
So currently in this code, in `message.actionWitness` is always defined. For testing in EVM it is handy to also make this the default `new ActionWitness` in `message.ts` - is this ok?

Reply:
yes i don't see the issue but probably this could be removed, lemme try removing it